synapse/tests/test_lib_stormlib_hashes.py

```diff
@@ -22,3 +26,111 @@ async def test_stormlib_crypto_hashes(self):
                 digests = {k: binascii.hexlify(v).decode() for k, v in hashset.digests()}
 
                 self.eq(hashes, digests)
+
+    async def test_stormlib_crypto_hmac(self):
+
+        async with self.getTestCore() as core:
+
+            q = '''$key=$key.encode() $mesg=$mesg.encode()
+            return ( $lib.crypto.hmac.sign(key=$key, mesg=$mesg, digest=$mode) )
```

Comment:
I kind of think this may be better suited as ``$lib.crypto.hashes.hmac()`` ?

Reply:
Yeah I think that fits in better

Reply:
I def would not consider HMAC "a hash algo" though. We should probably discuss.